include/openthread/instance.h

```diff
@@ -193,6 +193,17 @@ otInstance *otInstanceInit(void *aInstanceBuffer, size_t *aInstanceBufferSize);
 otInstance *otInstanceInitSingle(void);
 
 /**
+ * This function indicates whether or not the instance is valid, i.e., it has been initialized and not yet finalized.
+ *
```

Comment:
Added some more details in new commit...Let me know if this was what you had in mind...(feel free to suggest changed/edits... :) ...)